Dungeoneer/src/com/interrupt/dungeoneer/entities/Player.java

```diff
@@ -2591,5 +2591,30 @@ public void resetInventoryDrawables() {
 			if(itm != null && itm.drawable != null)
 				itm.drawable.refresh();
 		}
-    }
-}
\ No newline at end of file
+	}
+	
+	/** Returns the current player gold amount. */
+	public int getGoldAmount() {
+		return this.gold;
```

Comment:
Can we also normalize the white space while we are in this file? 😄

Reply:
Not sure what you mean. I set my settings to use four spaces, but I do not get any changes listed.